src/components/dropdown/Dropdown.tsx

```diff
@@ -0,0 +1,73 @@
+import { DropdownMenu } from '../dropdownMenu/DropdownMenu';
+import React from 'react';
+import { Person } from '../../types/Person';
+import debounce from 'lodash.debounce';
+
+type Props = {
+  onQueryChange: React.Dispatch<React.SetStateAction<string>>;
+  onPersonSelect: React.Dispatch<React.SetStateAction<Person | null>>;
+  people: Person[];
+  isEmptyPeople: boolean;
+  debounceDelay: number;
+};
+
+export const Dropdown: React.FC<Props> = ({
```

Comment:
You can import `FC` from React as well, so `React.` part can be omitted when using